roundtrip/RoundTrip/Spec.hs

```diff
@@ -146,7 +147,10 @@ roundTripSpec = do
       -- IPC
       hPutStrLn hin jsonReq
       err <- hGetLine herr
-      assertEqual "hs> Purescript shouldn't report an error" "" err
+      when (err /= "") $ do
```

Comment:
```suggestion
      unless (null err) $ do
```

Reply:
I prefer the `when`, somehow `unless` breaks my feeble mind